src/index.html

```diff
@@ -1,13 +1,253 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&family=Roboto&display=swap"
+      rel="stylesheet">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+
+  <body class="page__body">
+    <header class="header">
+      <div class="conteiner">
+        <div class="header__content">
+          <div class="top-actions">
+            <a href="/" class="header__logo">
+              <img
+                src="./images/logo--miami.svg"
+                alt="MyBIKE"
+                class="logo__img"
+              >
+            </a>
+
+            <a
+              href="tel:+1 234 5555-55-55"
```

Comment:
Don't use dashes and spaces at the href attribute for phone numbers to avoid errors while redirecting